orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -418,13 +418,23 @@ def keep_n_changed(self):
             self.frequency_filter.keep_n = self.keep_n
             self.change_signal.emit()
 
+    def _read_file(self, method, path):
+        self.master.Error.encoding.clear()
```

Comment:
This doesn't work when reloading schemas. If the error is raised when reading the stopwords file, it is always cleared when reading lexicon file.

Reply:
The same problem happens in another case. I open the broken file, and the error is shown. Then  I click on the lexicon dropdown and re-select (none). The error is cleared but the broken stopwords file is still selected.

Perhaps, should we have separate errors for stopwords and lexicon file?

Reply:
See no way around it.

Reply:
It's not pretty, but I don't see a simple way about it.

Fixed.